### PR TITLE
56 Scrolling Content

### LINKED ISSUE #56: Design QA notes - Scrolling
@huntermacd There are some scrolling issues that we'd like to get it fixed!

The Screenshot shows how we can scroll the entire page, instead of having the scrollable area within the text.
Screenshot 2024-08-09 at 3 07 41 PM

We'd like to have the image on the right be fixed from top to bottom, same with the navigation on the left side. But the text is the only scrollable content from the screen.

Here's a video of Jared's prototype showing the ideal scrolling.
https://github.com/user-attachments/assets/5da8d729-cfcb-4404-95e5-eb33d7ac0764

### PR DESCRIPTION
**Pull request by huntermacd**
Resolves #56 

This PR ensures only the main text area scrolls and not the sidebar or image containers. Additionally, updates dependencies and adds links to each available ebook when running the app on a local, since the bare server url now automatically redirects away from where you need to be.

To test:

- visit https://pr57-hwex9jpj3mqbvksmb6wq5oyglugldama.tugboatqa.com/ebooks/large-team-challenges/cover/
- page through until reaching a chapter with an image
- ensure that only the main center text area scrolls and not the sidebar or image container

### CODE REVIEW

**Comment by mark-casias** [comment]
There is some wonky action in Safari on chapter 8 (https://pr57-hwex9jpj3mqbvksmb6wq5oyglugldama.tugboatqa.com/ebooks/large-team-challenges/chapters/chapter-8/)

image

**Comment by mark-casias** [comment]
Confirmed fixed on safari, Brave and mobile Chrome

File: devMsg.js
Comment (sirkitree) [inline devMsg.js:1]: interesting workaround!

**Comment by sirkitree** [comment]
lgtm! :shipit:

**Pull request merged**